src/main.ts

```diff
@@ -96,7 +116,8 @@ async function main() {
         `/subscriptions/${taskParams.subscriptionId}/resourceGroups/${taskParams.resourceGroup}/providers/Microsoft.App/managedEnvironments/${managedEnvironmentName}`,
```

Comment:
Looks like it's enough to re-use `containerAppProperty.managedEnvironmentId` as `managedEnvironmentId` in `containerAppEnvelope` because the managed environmnt that is used by the container app is not be needed to change.